src/o365/spo/commands/feature/feature-list.ts

```diff
@@ -0,0 +1,175 @@
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${siteAccessToken}`,
+            accept: 'application/json;odata=nometadata'
+          }),
+          json: true
+        };
+
+        return request.get(requestOptions);
+      })
+      .then((features: { value: Feature[] }): void => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(features);
+          cmd.log('');
+        }
+
+        if (features.value && features.value.length > 0) {
+          if (args.options.output === 'json') {
+            cmd.log(features.value);
+          }
+          else {
+            cmd.log(features.value.map(f => {
+              return {
+                DefinitionId: f.DefinitionId,
+                DisplayName: f.DisplayName
+              };
+            }));
+          }
+        }
+        else {
+          if (this.verbose) {
+            cmd.log('No features found');
+          }
+        }
+        cb();
+      }, (err: any): void => this.handleRejectedPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --url <url>',
+        description: 'Url of the site (collection) to retrieve the feature from'
+      },
+      {
+        option: '-s, --scope [scope]',
+        description: 'Scope of the feature. Allowed values Site|Web. Default Web',
+        autocomplete: ['Site', 'Web']
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+
+      if (SpoCommand.isValidSharePointUrl(args.options.url) !== true) {
+        return 'Missing required option url';
+      }
+
+      if (args.options.scope) {
+        if (args.options.scope !== 'Site' &&
+          args.options.scope !== 'Web') {
+          return `${args.options.scope} is not a valid feature scope. Allowed values are Site|Web`;
+        }
+      }
+
+      return true;
+    };
+  }
+
+  public commandHelp(args: CommandArgs, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(commands.FEATURE_LIST).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} before using this command, log in to a SharePoint Online site,
+        using the ${chalk.blue(commands.LOGIN)} command.
+                      
+  Remarks:
+  
+    To retrieve list of features, you have to first log in to a SharePoint Online site using the
+    ${chalk.blue(commands.LOGIN)} command, eg. ${chalk.grey(`${config.delimiter} ${commands.LOGIN} https://contoso.sharepoint.com`)}.
+    When using the text output type (default), the command lists only the values of the ${chalk.grey('DefinitionId')},
+    and ${chalk.grey('DisplayName')} properties of the feature. When setting the output
+    type to JSON, all available properties are included in the command output.
```

Comment:
This is incorrect, because the REST request explicitly retrieves only DefinitionId and DisplayName. I noticed, that if we remove the explicit select, then you get back only the DefinitionId. With that, let's remove this remark and JSON output formatting altogether and always show DisplayName and DefinitionId for now.